.github/workflows/build-main.yaml

```diff
@@ -58,72 +58,75 @@ jobs:
-#          echo ""
-#          echo "tag      : ${{ github.ref_name }}"
-#          echo "version  : ${CONNECTORS_VERSION}"
-#          echo "revision : ${CONNECTORS_REVISION}"
-#          echo "release  : ${CONNECTORS_RELEASE}"
-#          echo "branch   : ${BRANCH_NAME}"
-#          echo ""
-#
-#          cd ${{ env.PROJECTS }}/cos-manifests
-#
-#          ls -lR connectors/cos-fleet-catalog-camel
-#
-#          git config user.email "mas-connectors@redhat.com"
-#          git config user.name "mas-connectors"
-#
-#          git checkout -b ${BRANCH_NAME}
-#          git add connectors/cos-fleet-catalog-camel
-#          git status
-#
-#          if [ -z "$(git status --untracked-files=no --porcelain connectors/cos-fleet-catalog-camel)" ]; then
-#            echo "Working directory clean excluding untracked files"
-#          else
-#            git commit -m "Update cos-fleet-catalog-camel " connectors/cos-fleet-catalog-camel
-#            git reset --hard
-#            git push -u origin ${BRANCH_NAME}
-#
-#            # GH CLI can't find the branch on remote... needs some time :)
-#            sleep 15
-#
-#            gh config set prompt disabled
-#
-#            gh pr create \
-#              --fill \
-#              --base main \
-#              --title "chore(catalog): update cos-fleet-catalog-camel to ${{ env.CONNECTORS_RELEASE }}" \
-#              --body "sha: ${{ github.sha }}, tag: ${{ github.ref_name }}"
-#          fi
+
+      - name: Login to Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Push Container Images
+        run: |
+          echo "Pushing container images"
+          echo "Pushing container images"
+          
+          IMAGES=$(docker image ls --format "{{.Repository}}:{{.Tag}}" | grep quay.io/sco1237896/connector- | grep ${CONNECTORS_RELEASE} || :;)
+          
+          echo "Images[@]: ${IMAGES[@]}"
+          echo "Images: ${IMAGES}"
+          
+          for IMAGE in ${IMAGES}; do
+              echo "Pushing image: ${IMAGE}"
+              docker push $IMAGE
+          done
+
+      - name: "Create PR for sco-connector"
+        env:
+          BRANCH_NAME: "sco-connector.${{ env.CONNECTORS_RELEASE }}"
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ""
+          echo "tag      : ${{ github.ref_name }}"
+          echo "version  : ${CONNECTORS_VERSION}"
+          echo "revision : ${CONNECTORS_REVISION}"
+          echo "release  : ${CONNECTORS_RELEASE}"
+          echo "branch   : ${BRANCH_NAME}"
+          echo ""
+
+          cd ${{ env.PROJECTS }}/sco-manifests
+
+          ls -lR connectors
+
+          git config user.email "sco-connectors@sco1237896.github.com"
+          git config user.name "sco-connectors"
+
+          git checkout -b ${BRANCH_NAME}
+          git add connectors/sco-connector
```

Comment:
The pipeline fails here with `fatal: pathspec 'connectors/sco-connector' did not match any files`. 

The `connectors` directory at that point has the following contents:

```
| drwxr-xr-x. 1 root root     544 Sep  6 07:16 connector-catalog-aws
| drwxr-xr-x. 1 root root     432 Sep  6 07:17 connector-catalog-azure
| drwxr-xr-x. 1 root root     334 Sep  6 07:19 connector-catalog-gcp
| drwxr-xr-x. 1 root root      28 Sep  6 07:19 connector-catalog-itops
| drwxr-xr-x. 1 root root     262 Sep  6 07:19 connector-catalog-messaging
| drwxr-xr-x. 1 root root      58 Sep  6 07:20 connector-catalog-misc
| drwxr-xr-x. 1 root root     228 Sep  6 07:21 connector-catalog-nosql
| drwxr-xr-x. 1 root root     400 Sep  6 07:22 connector-catalog-saas
| drwxr-xr-x. 1 root root     164 Sep  6 07:22 connector-catalog-social
| drwxr-xr-x. 1 root root     452 Sep  6 07:23 connector-catalog-sql
| drwxr-xr-x. 1 root root     220 Sep  6 07:23 connector-catalog-storage
| -rw-r--r--. 1 root root 1919348 Sep  6 07:23 connectors.json
```

Looking at the old cos-manifests repo, I assume the intention here is to have a subdirectory that would contain all the connector categories?

Reply:
for rhoc we had subdirectory for camel and debezium conenctors, here we don't so there is not more need to have a subdirectory